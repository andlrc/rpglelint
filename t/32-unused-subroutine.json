[
   {
      "column":7,
      "file":"t/32-unused-subroutine.rpgle",
      "line":9,
      "message":"'shouldFail' defined but not used",
      "severity":"warning"
   }
]
