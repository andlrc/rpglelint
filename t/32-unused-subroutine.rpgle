// rpglelint: -Wunused-variable

exsr shouldWork;

begsr shouldWork;
  return 'should work';
endsr;

begsr shouldFail;
  return 'should fail';
endsr;
