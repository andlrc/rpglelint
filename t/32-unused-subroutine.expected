t/32-unused-subroutine.rpgle:9:7: warning: 'shouldFail' defined but not used [-Wunused-subroutine]
 begsr shouldFail;
       ^
